tools/build_rules/verifier_test/cc_indexer_test.bzl

```diff
@@ -248,18 +252,18 @@ def _cc_extract_kzip_impl(ctx):
     ])
     outputs = depset([
         extract(
+            srcs = depset([src]),
```

Comment:
Did you run buildifier over these? If not, they will merely be undone the next time someone else edits them.

Reply:
I did.  Everything looks good on my end with both the internally built `buildifier` as well as with `get get -u github.com/bazelbuild/buildtools/buildifier`.